llvm/lib/Target/PowerPC/PPCExpandISEL.cpp

```diff
@@ -32,14 +32,7 @@ STATISTIC(NumExpanded, "Number of ISEL instructions expanded");
 STATISTIC(NumRemoved, "Number of ISEL instructions removed");
 STATISTIC(NumFolded, "Number of ISEL instructions folded");
 
-// If -ppc-gen-isel=false is set, we will disable generating the ISEL
-// instruction on all PPC targets. Otherwise, if the user set option
-// -misel or the platform supports ISEL by default, still generate the
-// ISEL instruction, else expand it.
-static cl::opt<bool>
-    GenerateISEL("ppc-gen-isel",
-                 cl::desc("Enable generating the ISEL instruction."),
-                 cl::init(true), cl::Hidden);
```

Comment:
As a first step, why not just set the default value to false here?

If we don't get any regressions, then as a follow up, we can remove the pass?

Reply:
The option is controlling whether to generate `isel` instruction rather than enabling this pass. I've moved this option to `PPCInstrInfo` where in my view is a proper place.